reference/halyard/custom.md

```diff
@@ -42,6 +42,18 @@ information.
 You can validate that your Custom Profiles have been picked up and applied to
 their services by checking `~/.hal/$DEPLOYMENT/history/service-profiles.yml`.
 
+### Custom Profile for Deck
+
+To supply custom settings or new values that override generated settings for the
+Deck service, you can place them in a file called `settings-local.js`. For
```

Comment:
s/service/UI/
(just to make it clear what Deck is, for newer users who might not know)